controller/src/schemas/session.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2023 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from enum import Enum
+from typing import List, Optional, Tuple
+
+from pydantic import BaseModel
+
+from controller.src.schemas.base import BaseWithOwner
+
+
+class QueryItem(BaseModel):
+    question: str
+    session_name: Optional[str] = None
+    filter: Optional[List[Tuple[str, str]]] = None
+    data_source: Optional[str] = None
```

Comment:
?

Reply:
This is the body of the query that goes into the infer workflow endpoint, how the data comes from the front-end now.